### PR TITLE
Support JSON config files

### PR DESCRIPTION
**Pull request by luca-drf**

### CODE REVIEW

File: ensembl_prodinf/config.py [hunk 1]
```diff
@@ -12,15 +13,18 @@ def load_config_yaml(file_path, strict=False):
         return config if config else {}
     else:
         if strict:
-            raise ValueError('Invalid config file path: {}'.format(file_path))
+            raise ValueError('Invalid config file path: %s' % file_path)
         else:
-            logger.warning('Using default configuration. Config file path was: {}'.format(file_path))
+            logger.warning('Using default configuration. Config file path was: %s', file_path)
```
Comment (marcoooo) [inline ensembl_prodinf/config.py:18]: For my personal knowledge, what's the gain here? I thought `format` was more 3.x-ish than `%s` notation.

Comment (luca-drf) [reply ensembl_prodinf/config.py:18]: It's good practice to use lazy interpolation with logging (i.e. the arguments gets evaluated only if the logger level is equal or above the level of the call. See: https://docs.python.org/3.5/library/logging.html#logging.debug)

In general, in terms of performance, `.format` is sensibly slower than `%` as the latter works at "interpreter level" (it doesn't require a function call etc.). 
See: https://stackoverflow.com/a/48465349

Comment (marcoooo) [reply ensembl_prodinf/config.py:18]: Ok. Thank you, see i thought this was the other way around in term of rapidity. And I even remember reading something about the % notation obsolescence. Must have been mistaken.

**Review by marcoooo — APPROVED** [review]

**Review by thomasmaurel — APPROVED** [review]

**Comment by thomasmaurel** [comment]
Sorry @radome: I clicked too quickly and merged your PR...

**Pull request merged**